draft-ietf-tls-external-psk-guidance.md

```diff
@@ -429,6 +429,11 @@ internal session resumption cache. This means that if a PSK identity collision o
 the application's external PSK usage will typically take precedence over the internal
 session resumption path.
 
+Since resumption PSK identity is assigned by the TLS stack implementation,
```

Comment:
I think either "the resumption PSK identity is assigned" (add "the") or "resumption PSK identities are assigned" (pluralize), but we can probably count on the RFC Editor to catch that.

Reply:
```suggestion
Since resumption PSK identities are assigned by the TLS stack implementation,
```